### PR TITLE
docs(api): add documentation for custom labware handling

### PR DESCRIPTION
**Pull request by sfoster1**

### CODE REVIEW

File: api/docs/v2/new_labware.rst [hunk 1]
```diff
@@ -26,14 +26,24 @@ A third optional argument can be used to give the labware a nickname to be displ
...
+Default Labware
+^^^^^^^^^^^^^^^
+
+The OT-2 has a set of labware well-supported by Opentrons defined internally. This set of labware is always available to protocols. This labware can be found on the `Opentrons Labware Library <https://labware.opentrons.com>`_. You can copy the load names that should be passed to ``protocol.load_labware`` statements to get the correct definitions.
+
+Custom Labware
+^^^^^^^^^^^^^^
 
-    Copy and paste labware API names directly from the `Opentrons Labware Library <https://labware.opentrons.com>`_ to ensure your ``protocol.load_labware`` statements get the correct definitions.
+If you have a piece of labware that is not in the Labware Library, you can create your own definition using the `Opentrons Labware Creator <https://labware.opentrons.com/create/>`_. Before using the Labware Creator, you should read the introduction article `here <https://labware.opentrons.com>`_.
```
Comment (Laura-Danielle) [inline api/docs/v2/new_labware.rst]: Shouldn't the introduction article be linked to this: https://support.opentrons.com/en/articles/3136504-creating-custom-labware-definitions

Comment (sfoster1) [reply api/docs/v2/new_labware.rst]: Yeah those are placeholder links didn't know that existed yet

**Comment by sfoster1** [comment]
Blocked waiting for a support docs link for the custom labware process

File: api/docs/v2/new_labware.rst (hunk 1 above)
Comment (pantslakz) [reply api/docs/v2/new_labware.rst]: @SyntaxColoring can inform on final URL to correct article

**Pull request merged**